src/qiboml/operations/expectation.py

```diff
@@ -0,0 +1,91 @@
+"""Compute expectation values of target observables with the freedom of setting any qibo's backend."""
+
+from typing import List, Optional, Union
+
+import qibo
+from qibo.backends import construct_backend
+from qibo.config import raise_error
+
+from qiboml.operations.differentiation import parameter_shift
+
+
+def expectation_on_backend(
```

Comment:
```suggestion
def evaluate(
```
?

Or otherwise just `expectation`.